assets/scripts/oppia-player-0.0.0.js

```diff
@@ -244,9 +244,8 @@
     var versionString = explorationVersion ? '&v=' + explorationVersion : '';
     iframe.setAttribute('src', encodeURI(
       (oppiaNode.getAttribute('src') || currLoc) +
-      '/explore/' + oppiaNode.getAttribute('oppia-id') +
-      '?iframed=true&locale=en' + versionString +
-      '#' + TAG_ID_KEY + tagId +
+      '/embed/exploration/' + oppiaNode.getAttribute('oppia-id') +
```

Comment:
Please don't change this yet. In fact we can't change anything in assets/scripts... these scripts are served from elsewhere over a CDN and others are already using them.